common/amundsen_common/utils/atlas_utils.py

```diff
@@ -0,0 +1,245 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+import abc
+import re
+from typing import Any, Dict, Optional
+
+
+class AtlasStatus:
+    ACTIVE = "ACTIVE"
+    DELETED = "DELETED"
+
+
+class AtlasCommonParams:
+    qn = 'qualifiedName'
```

Comment:
why not `qualified_name`? makes it more readable. same for the rest of the properties in this class.

Reply:
ok, can rename them